pandas/_libs/lib.pyx

```diff
@@ -3050,6 +3050,27 @@ def is_bool_list(obj: list) -> bool:
     return True
 
 
+cpdef ndarray eq_NA_compat(ndarray[object] arr, object key):
```

Comment:
maybe move to missing.pyx? (follown)

Reply:
i was thinking indexes.pyx since thats the only place it is used, but will give some thought